ska_helpers/retry/api.py

```diff
@@ -21,6 +23,33 @@ def __init__(self, failures):
         self.failures = failures
 
 
+def _mangle_alert_words(msg):
+    """
+    Mangle alert words "warning", "error", "fatal", "fail", "exception" in a string.
+
+    This is done by replacing "i" or "l" with "1" and "o" with "0" in the middle of
+    any of these words. The intent is to avoid triggering the task schedule "check" for
+    for those words. This is done with a case-insensitive regex substitution.
+
+    Example::
+
+        >>> mangle_alert_words("WARNING: This is a fatal Error message.")
```

Comment:
In passing I note this example suggests mangle_alert_words is available as a function without the leading underscore.